webapp/views/partials/users.jade

```diff
@@ -1,4 +1,5 @@
 div(class='container', ng-if='init()')
+    -var selected = 'Users';
```

Comment:
Nah I think it should be in the jade, all it does is give the jade somewhat of a "state". So if it is on the users page, it will disallow you to click the users button.